### PR TITLE
GDB-8753 Implemented ACL management page

### PR DESCRIPTION
**Pull request by svilenvelikov**
## What
Implemented an ACL management page.

## Why
By requirement.

## How
* Implemented the page, services, tests.
* Recreated the package-lock.json due to inconsistencies.

### CODE REVIEW

File: src/i18n/locale-fr.json [hunk 1]
```diff
@@ -119,6 +119,31 @@
             "content": "Sélectionner les nœuds à retirer du cluster existant"
         }
     },
+    "acl_management": {
+        "rulestable": {
+            "column": {
+                "index": "#",
+                "reorder": "",
```
Comment (yordanalexandrov) [inline src/i18n/locale-fr.json:126]: What is this for?

File: src/js/angular/aclmanagement/controllers.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+    };
+
+    //
+    // Private functions
+    //
+
+    const loadRules = () => {
+        $scope.loading = true;
+        const repositoryId = getActiveRepository();
+        AclManagementRestService.getRules(repositoryId).success((response) => {
```
Comment (yordanalexandrov) [inline src/js/angular/aclmanagement/controllers.js]: Did we not talk at some point that it is better to use `.then` instead of `.success`?

Since they are depricated (not that we are going to update AngularJS, but still)

File: src/pages/aclmanagement.html [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+                                gdb-tooltip="{{'acl_management.rulestable.actions.move_down' | translate}}">
+                            <i class="fa fa-caret-down"></i>
+                        </button>
+                    </td>
+                    <td>{{rule.subject}}</td>
+                    <td>{{rule.predicate}}</td>
+                    <td>{{rule.object}}</td>
+                    <td>{{rule.context}}</td>
+                    <td>{{rule.role}}</td>
+                    <td>{{rule.policy}}</td>
```
Comment (yordanalexandrov) [inline src/pages/aclmanagement.html:72]: Shouldn't this also be translated? Or you want all data to be shown as is regardless if it is an enumeration of some kind?

File: test-cypress/fixtures/locale-en.json [hunk 4]
```diff
@@ -52,8 +52,8 @@
             "message_size_kb_tooltip": "Message size concerns the size of the data blocks transferred during data replication streaming through the RPC protocol.",
             "verification_timeout": "Verification timeout",
             "verification_timeout_tooltip": "Verification timeout concerns the amount of time a follower node would wait until attempting to verify last committed entry when the first verification is unsuccessful.",
-            "transaction_log_max_size_gb": "Transaction log maximum size",
-            "transaction_log_max_size_gb_tooltip": "Maximum size of the transaction log in GBs that will trigger an automatic truncation of the transaction log if it becomes bigger. The minimum transaction log size is 1 GB. Negative numbers will disable the automatic transaction log truncation."
+            "transaction_log_maximum_size_gb": "Transaction log maximum size GB",
```
Comment (yordanalexandrov) [inline test-cypress/fixtures/locale-en.json:55]: Shouldn't this be in a separate commit?

Comment (yordanalexandrov) [reply test-cypress/fixtures/locale-en.json:55]: Hm.. maybe a rebase on master is in order?

Comment (yordanalexandrov) [reply test-cypress/fixtures/locale-en.json:55]: This is just an update of the test fixture, so it is ok

File: src/pages/aclmanagement.html (hunk 3 above)
Comment (svilenvelikov) [reply src/pages/aclmanagement.html:72]: Exactly. This is data and is displayed as is. Pavel insisted that we don't want to represent it with some component like a checkbox for example.

File: src/i18n/locale-fr.json (hunk 1 above)
Comment (svilenvelikov) [reply src/i18n/locale-fr.json:126]: The label for the reorder actions column :)
Maybe I should remove it

File: src/js/angular/aclmanagement/controllers.js (hunk 2 above)
Comment (svilenvelikov) [reply src/js/angular/aclmanagement/controllers.js]: Yes. My bad. I tested something and forgot to revert this.

**Review by yordanalexandrov — APPROVED** [review]
